### DISCUSSION TITLE
Install Keras Macbook M1

### DISCUSSION THREAD

**Discussion by SholahShoya** [teamalgoritma/community#478] [Q&A] [answered]
<img width="969" alt="image" src="https://user-images.githubusercontent.com/112464016/206864324-afba11c2-a533-4f9f-ba85-96942c392009.png">

Untuk LBB NN + Tensorflow keras, saya ga berhasil install karena architecture m1 berbeda (seperti error yang dibilang diatas). architecture untuk keras menggunakan "x86_64" sementara architecture saya "arm64". sudah mencoba pake homebrew/rosetta tapi masih belum berhasil. mungkin team algoritma bisa bantu provide guide step-by-step bagaimana caranya agar keras bisa berfungsi di R saya? untuk installation bisa bebas apakah itu installation menggunakan base/homebrew/rosetta. thanks in advance

**Comment by nisa-basalamah** [comment]
Dear @SholahShoya,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima. Kami akan mencoba membantu mencarikan referensi berkaitan dengan pertanyaan yang diberikan.

**Reply by SholahShoya** [reply]
Thanks so much kak. Been working around this issue for 2 days straight with no luck.

**Comment by amaliaasf** [comment, marked as answer]
Dear @SholahShoya, kami sudah mencoba beberapa referensi terkait installasi keras dengan OS Mac Chip M1/M2, namun sampai saat ini kami belum berhasil menemukan solusi yang tepat. Untuk itu, saran dari kami sejauh ini Bapak bisa mencoba membuat model dengan keras menggunakan Kaggle. Untuk pembelajaran kelas NN, instruktur juga sudah melampirkan link kaggle berisikan inclass di Google Classroom.

Demikian yang dapat kami sampaikan, semoga membantu Pak :)
